src/core/hle/service/nwm/nwm_uds.cpp

```diff
@@ -581,6 +581,10 @@ void NWM_UDS::Shutdown(Kernel::HLERequestContext& ctx) {
 
     recv_buffer_memory.reset();
 
+    SharedPage::Handler& shared_page = Core::System::GetInstance().Kernel().GetSharedPageHandler();
```

Comment:
```suggestion
    SharedPage::Handler& shared_page = system.Kernel().GetSharedPageHandler();
```
ditto for all other cases

Reply:
This hasn't been resolved